explainer.md

```diff
@@ -48,218 +49,298 @@ Provide web apps with efficient access to built-in (software and hardware) media
   - control over buffer behavior
   - spatial and temporal scalability
 - Decoded and encoding images
-- Reencoding multiple input media streams in order to merge many encoded media streams into one encoded media stream.
+- Re-encoding multiple input media streams in order to merge many encoded media streams into one encoded media stream.
 
 
 ## Proposed solutions
 
-We build on top of [WHATWG Streams](https://streams.spec.whatwg.org/) (in particular [TransformStreams](https://streams.spec.whatwg.org/#ts-class)) and integrate with [MediaStreamTracks](https://www.w3.org/TR/mediacapture-streams/#dom-mediastreamtrack).
+The WebCodecs interface is modeled on well known platform and software codec APIs. This affords the full power of the underlying platform codecs and is intuitive for authors with experience in developing media applications.
```

Comment:
Nit: I don't think we plan to offer the 'full' power, as that would require exposing every platform detail.

Reply:
Agree. Not sure what I was thinking. I've re-worded to:

The WebCodecs interface is modeled on well known platform and software codec APIs. This API shape is time-tested and intuitive for authors with experience in developing media applications.